### DISCUSSION TITLE
Compilation albums

### DISCUSSION THREAD

**Discussion by xavix** [jcorporation/myMPD#654] [Ideas]
Let me first give my congratulations for the good work is myMPD. I'm really happy to play with it.
Just an idea about organizing the album collection. 
When the selected filter (in the upper left roller) is "Database / Album", those compilation albums (albums with various artists inside) imo should appear only once per album, not once per artist, with the albumartist tag instead the artist tag.
Thanks

**Comment by jcorporation** [comment]
This should already be the case. myMPD falls only back to Artist, if the AlbumArtist tag is not enabled.
